### PR TITLE
Convert ItUsabilityOperatorHelmChart#testDeleteOperatorButNotDomain to JUnit5

### PR DESCRIPTION
**Pull request by xiancao**
Convert ItUsabilityOperatorHelmChart#testDeleteOperatorButNotDomain to JUnit5 test framework.

### CODE REVIEW

**Comment by xiancao** [comment]
Jenkins test result: https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/72/
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-model-in-image-tests/650/

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+    int nodeportshttps = getNextFreePort(30443, 30543);
+
+    // install and verify NGINX
+    logger.info("Installing and verifying NGINX");
+    nginxHelmParams = installAndVerifyNginx(nginxNamespace, nodeportshttp, nodeportshttps);
+  }
+
+  /**
+   * Create operator and verify it is deployed successfully.
+   * Create domain1 and verify all server pods in domain1 were created and ready.
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:110]: "Create domain1"   does not make any sense. Better say "Create a Custom Domain Resource with DomanUid as domain1"  Here domain1 is a string, not a K8s Object.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+  @Slow
+  @MustNotRunInParallel
+  public void testDeleteOperatorButNotDomain() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    HelmParams opHelmParams = installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    createAndVerifyMiiDomain();
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:130]: It is not needed, the extension framework will take care. See the ItMiiDomain.java as a reference.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+
+    // verify the sample app in the domain is still accessible from all managed servers through NGINX
+    logger.info("Checking that the sample app can be accessed from all managed servers through NGINX "
+        + "after the operator was deleted");
+    verifySampleAppAccessThroughNginx();
+  }
+
+  /**
+   * TODO: remove this after Sankar's PR is merged
+   * The cleanup framework does not uninstall NGINX release. Do it here for now.
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:198]: Check with Sankar. I think his PR has been mergrd in develop

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+      // check that the managed server service was created
+      logger.info("Checking that managed server service {0} was created in namespace {1}",
+          managedServerPodName, domainNamespace);
+      checkServiceCreated(managedServerPodName, domainNamespace);
+    }
+  }
+
+  /**
+   * Create an ingress for the domain.
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:268]: Can it be an Action to be used by any ItClass

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java [hunk 5]
```diff
@@ -377,4 +379,42 @@ public static boolean adminServerReady(String domainUid, String namespace) {
     return true;
   }
 
+  /**
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java:382]: This method is alreday merged in develop. Please resolve with develop branch

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java (hunk 2 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:130]: this is a local method. I am using the MII_BASIC_IMAGE_NAME built by our framework. This method is create the MII domain using the pre-built image and verify the pods are ready.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java (hunk 3 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:198]: not yet.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java (hunk 1 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:110]: fixed

**Comment by xiancao** [comment]
Jenkins test result: https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/80/

**Pull request closed without merging**